lib/kube/proxy/forwarder.go

```diff
@@ -536,7 +541,7 @@ func (f *Forwarder) formatResponseError(rw http.ResponseWriter, respErr error) {
 	}
 }
 
-func (f *Forwarder) setupContext(ctx auth.Context, req *http.Request, isRemoteUser bool, certExpires time.Time) (*authContext, error) {
+func (f *Forwarder) setupContext(ctx auth.Context, req *http.Request, isRemoteUser bool, clientIdentity *tlsca.Identity) (*authContext, error) {
```

Comment:
While here, can we use another var name for the `auth.Context`? I prefer to keep `ctx` reserved for `context.Context`.